modules/test/helpers/common.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2022 Red Hat
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package helpers
+
+import (
+	"context"
+	"os"
+	"strconv"
+	"time"
+
+	ginkgo "github.com/onsi/ginkgo/v2"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// TestHelper -
+type TestHelper struct {
+	k8sClient client.Client
+	ctx       context.Context
+	cancel    context.CancelFunc
+	timeout   time.Duration
+	interval  time.Duration
+}
+
+// NewDefaultTestHelper returns a TestHelper with some defaults
+func NewDefaultTestHelper(
+	k8sClient client.Client,
+) *TestHelper {
+	ctx, cancel := context.WithCancel(context.TODO())
+
+	return &TestHelper{
+		k8sClient: k8sClient,
+		ctx:       ctx,
+		cancel:    cancel,
+		timeout:   time.Second * 10,
+		interval:  time.Millisecond * 200,
```

Comment:
I would make this configurable by passing in the values.

Reply:
this was meant to get a helper with defaults for timeout/interval. I'll rename it to NewTestHelper and make them as params

Reply:
done

Reply:
/me is sad about not having default parameters in golang.

Right now I'm not sure about what would be the right defaults. I'm actively fine tuning the nova defaults. So it is better to keep it configurable. When we have the default settled then we can move that default over to lib-common.

Reply:
sure, we can add then a func which initialize them with some good defaults.